src/lib.rs

```diff
@@ -899,7 +899,7 @@ macro_rules! commit_info {
     };
 }
 
-/// Obtains the [`CrossToml`] from one of the possible locations
+/// Obtains the [`CrossToml`][1] from one of the possible locations
 ///
 /// These locations are checked in the following order:
 /// 1. If the `CROSS_CONFIG` variable is set, it tries to read the config from its value
```

Comment:
Hmm, this shouldn't be a broken link, it's a [intra-doc link](https://doc.rust-lang.org/rustdoc/write-documentation/linking-to-items-by-name.html), please revert this